crates/bevy_text/src/lib.rs

```diff
@@ -71,6 +71,10 @@ use bevy_render::{
 };
 use bevy_sprite::SpriteSystem;
 
+/// The raw data for the default font used by `bevy_text`
+#[cfg(feature = "default_font")]
+pub const DEFAULT_FONT_DATA: &[u8; 18848] = include_bytes!("FiraMono-subset.ttf");
```

Comment:
```suggestion
pub const DEFAULT_FONT_DATA: &[u8] = include_bytes!("FiraMono-subset.ttf");
```

I don't think hardcoding the amount of bytes is a good idea in this case: if the font is ever modified in the future, this will have to be updated.

Beyond that, I'm totally in favor of this!

Reply:
You can't have an unsized const.